plugins/github-pull-requests-board/src/utils/types.tsx

```diff
@@ -31,12 +31,36 @@ export type GraphQlPullRequests<T> = {
   };
 };
 
+export type GraphQlUserPullRequests<T> = {
+  user: {
+    pullRequests: {
+      edges: T;
+      pageInfo: {
+        hasNextPage: boolean;
+        endCursor?: string;
+      };
+    };
+  };
+};
+
 export type PullRequestsNumber = {
   node: {
     number: number;
   };
 };
 
+export type PullRequestsNumberAndOwner = {
+  node: {
+    number: number;
+    repository: {
```

Comment:
at this point this could have been `repository: Repository` right

Reply:
Right, good catch, will update